src/Tempest/Http/src/RouteConfig.php

```diff
@@ -14,12 +16,15 @@ final class RouteConfig
     /** @var array<string, string> */
     public array $matchingRegexes = [];
 
+    public RoutingTree $routingTree;
+
     public function __construct(
         /** @var array<string, array<string, \Tempest\Http\Route>> */
         public array $staticRoutes = [],
         /** @var array<string, array<string, \Tempest\Http\Route>> */
         public array $dynamicRoutes = [],
     ) {
+        $this->routingTree = new RoutingTree();
```

Comment:
At this point I'm wondering whether the routing tree will be compiled and cached somewhere?

Reply:
No , it will not. Because of `__sleep` code. I do not return "routingTree" so it shouldn't be serialized.